qcs/src/qpu/quilc/isa/mod.rs

```diff
@@ -99,7 +120,7 @@ mod describe_compiler_isa {
                         second_f64,
                         F64Margin {
                             ulps: 1,
-                            epsilon: 0.0000001
+                            epsilon: 0.000_000_1
```

Comment:
There are a bunch of these adding separators to large constants... they're just here to get rid of excess red squiggles from my IDE so it's easier to see and fix real errors.

Allegedly that warning comes from Clippy but running Clippy didn't report them so 🤷